lib/model/init.js

```diff
@@ -1,9 +1,11 @@
-const getOutputString = (username, password) => {
-  return new Promise( (resolve, reject) =>
-    resolve(`Your username is: ${username}\nYour password is: ${password}`)
-  );
-}
+const chalk = require('chalk');
```

Comment:
model modules are not to be responsible for view / presentation / format tasks. It is better to export an object out of the lib/model/init.js module which will be used by the view to format and present.
The pattern of exporting data objects from model to views is pretty common. Please change the code accordingly.